analysers/analyser_osmosis_powerline.py

```diff
@@ -326,22 +326,26 @@ def __init__(self, config, logger = None):
 '''The line ends in a vacuum, and should be connected to another line or
 a transformer `power=substation` or a generator `power=generator`.'''))
         self.classs[6] = self.def_class(item = 7040, level = 3, tags = ['power', 'fix:imagery'],
-            title = T_('Unfinished power minor line'))
+            title = T_('Unfinished power minor line'),
+            detail = T_(
+'''The line ends in a vacuum, and should be connected to another line or
+a transformer `power=substation` or a generator `power=generator`.'''))
         self.classs[3] = self.def_class(item = 7040, level = 3, tags = ['power', 'fix:chair'],
             title = T_('Connection between different voltages'),
             detail = T_(
 '''Two lines meet in one point, but inconsistent with voltages from the
 tag `voltage=*`.'''))
         self.classs_change[4] = self.def_class(item = 7040, level = 3, tags = ['power', 'fix:imagery'],
             title = T_('Non power node on power way'),
+            detail = T_('Power lines can only form a straight line from point to point and therefore shouldn\'t have additional nodes that aren\t supports or endpoints.'),
```

Comment:
This is not clear.

from *pole* to *pole*

Missing a quote.

Can you wrap the text like others, please.